test/output-consistency/mzcompose.py

```diff
@@ -0,0 +1,37 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from materialize.mzcompose import Composition, WorkflowArgumentParser
+from materialize.mzcompose.services import Cockroach, Materialized
+from materialize.output_consistency.output_consistency import (
+    parse_output_consistency_input_args,
+    run_output_consistency_tests,
+)
+
+SERVICES = [
+    Cockroach(setup_materialize=True),
+    Materialized(propagate_crashes=True, external_cockroach=True),
+]
+
+
+def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
+    """
+    Test the output consistency of different query evaluation strategies (e.g., dataflow rendering and constant folding).
+    """
+
+    c.down(destroy_volumes=True)
+
+    c.up("materialized")
+
+    args = parse_output_consistency_input_args(parser)
+    connection = c.sql_connection()
+
+    test_summary = run_output_consistency_tests(connection, args)
+
```

Comment:
Maybe print the summary here in a `finally` block?

Reply:
I think this is the only comment I haven't addressed...

What would you like to achieve by that? The summary is printed from within `run_output_consistency_tests` (or somewhere deeper). Just adding a finally block will not work because `test_summary` won't be in scope there.